client/js/views/templates/Header.js

```diff
@@ -1,14 +1,11 @@
 module.exports = function( { model } ) {
-const navOptions = model.forEach( datum => `<span>${this.CapitalizeFirstLetter(datum)}</span>` )
-return `<nav>
-    <div>
-        <div>
-            <span>Future</span>
-            <span>Days</span>
-        </div>
-        <div>Software</div>
-    </div>
-    <div data-js="navList">${navOptions}</div>
-</nav>
-`
+	const navOptions = model.forEach(datum => `<span> ${ this.CapitalizeFirstLetter(datum) } </span>`)
+	return `<nav>
+	<div class = "hDiv"> <a class = "hLink" href = "InsertURLToAboutUS"> ABOUT US </a> </div>
+	<div class = "hDiv"> <a class = "hLink" href = "InsertURLToWhereToFindUs"> WHERE TO FIND US </a> </div>
+	<div class = "hDiv"> <a class = "hCurrent" href = "InsertURLToFutureDaysFarm"> FUTURE DAYS FARM </a> </div>
+	<div class = "hDiv"> <a class = "hLink" href = "InserURLToTheBlog"> THE BLOG </a> </div>
+	<div class = "hDiv"> <a class = "hLink" href = "InsertURLToOurOfferings"> OUR OFFERINGS </a> </div>
+	<div class = "hLogo"> <img class = "hImg" src = "https://storage.googleapis.com/five-gallon/FutureDaysFarmLogo.svg" /> </div>
+	</nav>`
```

Comment:
Instead of typing all of these options by hand, you can utilize a model.

```
return `<nav>${navOptions}`
```

No need to utilize classes for each item, just use parent > child selectors.
Also, we can utilize `navigate` events in the view in order to route to a new page, no need to have the browser reload the page.
Please do not use all caps in the HTML, use css `text-transform: uppercase` to do this for you.
Omit all of the whitespace in the HTML ( except for newlines )
```
const navOptions = model.forEach( datum => {
   if( datum.type = 'logo' ) return `<img src=${this.ImageSrc('logo')} />`
   else {
        return <div>${datum.label}</div>
   }
} )
```

Then use the view to bind click events.  See Hazy `ecommerce` branch, or chat me when you get to this part.